### DISCUSSION TITLE
issues with annotation

### DISCUSSION THREAD

**Discussion by troutena** [stuart-lab/signac#773] [Q&A] [answered]
Hi All!

My name is Allison and I am a newb to analyzing single cell data. I am working through the Seurat and Signac vignettes that the Sajita lab provides but I am worried about an error code affecting downstream analysis.

For background, my data is from a 10x multiome run to give me scRNA and scATAC sequencing data. There are a few warnings that I'm worried may be affecting downstream commands. I basing my code of the pbmc vignette and my script is attached.
[script.odt](https://github.com/timoast/signac/files/7100899/script.odt)

Everything runs well until command:

`annotations <- GetGRangesFromEnsDb(ensdb = EnsDb.Mmusculus.v79)`

from which I get 21 warnings similar to this:

**In .Seqinfo.mergexy(x, y) :
  The 2 combined objects have no sequence levels in common. (Use
  suppressWarnings() to suppress this warning.)**

For background, I used refdata-cellranger-arc-mm10-2020-A-2.0.0 (provided on the 10x website) as my reference genome when running Cell Ranger ARC. Is this genome compatible with Ensembl genomes for annotation? I know there is a way to build a custom reference and then I could ensure compatibility. The annotations file that the code outputs looks normal, but I don't see any difference in the ATACob Seurat object when I try to annotate with the following code:

```
ATACob
granges(ATACob)
Annotation(object = ATACob) <- annotations
ATACob
```

Any help you could provide is very much appreciated. By the way I am using Seurat 4.0.4 and Signac 1.3.0 on R 4.1.0.

Thanks!

-Allison

**Comment by timoast** [comment, marked as answer]
Hi @troutena, those warnings can be ignored, I don't think there's any issue here

**Reply by troutena** [reply]
Hello @timoast thank you so much!

I think the error is in successfully annotating the Seurat object because later in the script when I run:
```

celltype.predictions <- TransferData(anchorset = transfer.anchors, refdata = GEXob$Annotations,
                                     weight.reduction = ATACob[["lsi"]], dims = 2:30)

```
I get the error: 

_Error: Cannot find 'Annotations' in this Seurat object_

And when I perform the head function on GEXob, I don't see any annotation column. I see orig.ident, nCount_RNA, and nFeature_RNA. Do you think the annotation function isn't working?


Thanks!

-Allison

**Reply by timoast** [reply]
Hi Allison, this is not related to gene annotations stored in the ChromatinAssay object. Here `GEXob$Annotations` is referring to metadata associated with each cell in the `GEXob` object. The error indicates that there is no such metadata named "Annotations" in the Seurat object. I'd suggest looking over the code you're using to process the Seurat object and double check that the expected metadata is being added. If you have further questions, you can open an issue in the Seurat repository since this relates to Seurat functionality.
